scripts/compose.py

```diff
@@ -468,8 +468,8 @@ def render_proxy(self):
 class Elasticsearch(StackService, Service):
     default_environment = ["cluster.name=docker-cluster", "bootstrap.memory_lock=true", "discovery.type=single-node"]
     default_es_java_opts = {
-        "-Xms": "1g",
-        "-Xmx": "1g",
+        "-Xms": "5g",
+        "-Xmx": "5g",
```

Comment:
Won't this be too large of a default for most people?  Default Docker for mac setting is 2 GB memory + 1 GB swap, I'm not sure if these settings will cause an immediate problem though.

I can commit to implementing https://github.com/elastic/apm-integration-testing/blob/9497bb2a70a9c449219b207e7126e5277f200ae0/scripts/compose.py#L483 this week, in case that affects your response.

Reply:
I'm currently modifying the compose.py file, so having command line options for this would be great!

I'll leave the default as-is.